homeassistant/components/nmap_tracker/config_flow.py

```diff
@@ -126,7 +129,7 @@ def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
         """Initialize options flow."""
         self.options = dict(config_entry.options)
 
-    async def async_step_init(self, user_input=None):
+    async def async_step_init(self, user_input=None) -> FlowResult:
```

Comment:
Missing typing of `user_input`.
```suggestion
    async def async_step_init(self, user_input: dict[str, Any] = None) -> FlowResult:
```

Reply:
Thanks 👍